analytics/Dockerfile

```diff
@@ -0,0 +1,122 @@
+# Use the official python3 image based on Debian 11 "Bullseye".
+# https://hub.docker.com/_/python
+
+# The build stage that will be used to deploy to the various environments
+# needs to be called `release` in order to integrate with the repo's
+# top-level Makefile
+FROM python:3-slim AS base
+
+# Install poetry, the package manager.
+# https://python-poetry.org
+RUN pip install --no-cache-dir poetry --upgrade
+
+RUN apt-get update \
+  # Install security updates
+  # https://pythonspeed.com/articles/security-updates-in-docker/
+  && apt-get upgrade --yes \
+  && apt-get install --no-install-recommends --yes \
+  build-essential \
+  libpq-dev \
+  postgresql \
```

Comment:
Some of these apt-get values might be removable like this one if the analytics code never hits postgres

Reply:
- build-essential: a good all around package to keep
- libpq-dev: is for postgres
- PostgreSQL: same as above

So, nothing to remove

Reply:
It doesn't right now, but it will need to in the future when we set up the load to Aurora for our data warehouse, so might not be terrible to leave it in for now